Makefile

```diff
@@ -0,0 +1,30 @@
+.PHONY: build
+build:
+	mkdir -p bin && export CGO_ENABLED=0 GOOS=linux && go build -o bin/app -ldflags '-s -w' ./core/main.go
+
+.PHONY: swagger-build
+swagger-build:
```

Comment:
‘swagger’ perhaps? 'swagger' below should be 'swagger-run', keep up with ‘horizon’ command.

Reply:
changed